THE_PLAN.md

```
@@ -22,53 +27,45 @@
+| PR 4 | refactor(canary): consume eslint-summary, delete bespoke tools (eslint-config-voxpelli) | ⬜ | PR 3 published | —  | n/a     |
 
-Legend: ⬜ not started · 🟡 in progress · 🟠 in review · ✅ merged · 🔴 blocked
+Legend: ⬜ not started · 🟡 in progress · 🟠 in review · ✅ merged · ❌ dropped · 🔴 blocked
 
 ## PR detail
 
-### PR 1a — fix: harden markdown output against injection
-- Branch: `fix/markdown-injection-hardening`
-- AC: npm run check + test:node + test:real-world green; new fixtures for `</details>` / `javascript:` / HTML-special; CHANGELOG auto-generated
-- Notes:
+### PR 3 — feat: add eslint-summary CLI (prepare + aggregate)
```

markdownlint-cli2 (0.23.2)

[warning] Headings should be surrounded by blank lines
Expected: 1; Actual: 0; Below

(MD022, blanks-around-headings)